x-pack/plugins/cases/server/saved_object_types/migrations/configuration.test.ts

```diff
@@ -99,7 +99,7 @@ describe('configuration migrations', () => {
     it('does not modify the other attributes of the saved object', () => {
       const config = createConnectorAttributeMigration(create_7_9_0_configSchema());
 
-      const configAttributes = config as SavedObjectSanitizedDoc<CasesConfigureAttributes>;
+      const configAttributes = config as SavedObjectSanitizedDoc<ConfigurationAttributes>;
```

Comment:
nit: As we are in the server folder maybe it is better to use `ConfigurationTransformedAttributes`. Same for the files below.